src/main/groovy/com/github/benmanes/gradle/versions/updates/Resolver.groovy

```diff
@@ -65,7 +67,7 @@ class Resolver {
     this.projectUrls = new ConcurrentHashMap<>()
     this.resolutionStrategy = resolutionStrategy
     this.project = project
-    this.checkConstraints = checkConstraints;
+    this.checkConstraints = checkConstraints
 
     useSelectionRules = new VersionComparator(project)
```

Comment:
I suppose these workarounds could go away if 5.0+ as a baseline (we started this plugin in 1.0 world!)

Reply:
Damn, that's ancient! 🤯 Does caa6aa2ac135aa848f8523f3dcd535c31af35c0f look good? And is there anything else we can remove?